src/snowflake/cli/plugins/object/stage/commands.py

```diff
@@ -121,3 +123,27 @@ def stage_diff(
     """
     diff: DiffResult = stage_diff(Path(folder_name), stage_name)
     return ObjectResult(str(diff))
+
+
+@app.command("execute", requires_connection=True)
+def execute(
+    stage_path: str = typer.Argument(
+        None,
+        help="Execute immediate all files from the path. Files can be filtered with glob like pattern, e.g. `@stage/**/*.sql`.",
+    ),
+    on_error: OnErrorType = typer.Option(
+        OnErrorType.BREAK.value,
+        "--on-error",
+        help="What to do when an error occurs. Defaults to break.",
+    ),
+    parameters: Optional[List[str]] = typer.Option(
+        None,
+        help='Parameters for the template. For example: `--parameters "<key>=<value>"`, string values should be in single quotes.',
+    ),
+    **options,
+):
+    """
+    Execute immediate all files from the stage path.
+    """
+    results = StageManager().execute(stage_path, on_error, parameters)
```

Comment:
Pass argument using keywords, it's easier to refactor later on

Reply:
Added